extensions/azurecore/src/azurecore.d.ts

```diff
@@ -38,11 +38,17 @@ declare module 'azurecore' {
 		azureAuthType?: AzureAuthType
 
 		providerSettings: AzureAccountProviderMetadata;
+
 		/**
 		 * Whether or not the account is a Microsoft account
 		 */
 		isMsAccount: boolean;
 
+		/**
+		 * Represents the tenant that the user would be signing in to. For work and school accounts, the GUID is the immutable tenant ID of the organization that the user is signing in to.
+		 * For sign-ins to the personal Microsoft account tenant (services like Xbox, Teams for Life, or Outlook), the value is 9188040d-6c67-4c5b-b112-36a304b66dad.
```

Comment:
Is the tenant for personal accounts not `f8cdef31-a31e-4b4a-93e4-5f571e91255a`?

Reply:
They both are valid, the one I mentioned in comment comes for MSAL ([MS Doc](https://learn.microsoft.com/en-us/azure/active-directory/develop/accounts-overview)). Since we're soon going to be in MSAL world (as we should be ideally), we can let the comment stay as is.